File: estrutura-de-dados/2021-05-20/Merge_Sort/python/main.py
from merge_sort import merge_sort


def main():
    array = [7, 15, 19, 21, 5, 9, 11, 90]
    print("Array antes: % s" % list(map(str, array)))
    merge_sort(array)
    print("Array depois: % s" % list(map(str, array)))


main()

File: estrutura-de-dados/2021-05-20/Merge_Sort/java/Main.java
import src.MergeSort;

class Main {
  public static void main(String[] args) {
    int[] array = new int[] { 15, 12, 67, 7, 15, 19, 21, 12, 55, 5, 9, 11, 90 };

    System.out.print("Array antes: ");
    for (int item : array) {
      System.out.print("|" + item + "|");
    }

    MergeSort.order(array);

    System.out.println("");

    System.out.print("Array depois: ");
    for (int item : array) {
      System.out.print("|" + item + "|");
    }
  }
}